examples/observability/eks-java-jmx/dev.tfvars

```diff
@@ -0,0 +1,2 @@
+grafana_api_key  = "<inset grafana api key>"
+grafana_endpoint = "https://<grafana endpoint>/"
```

Comment:
Add this to deployment README on what customers needs to do for this variable

Reply:
added